src/composables/useChatAPI.js

```
@@ -125,10 +125,31 @@ export function useChatAPI() {
         },
         body: JSON.stringify({
           question: question,
-          chat_history: chatHistory.map(msg => ({
-            ...msg,
-            sender: msg.sender === 'bot' ? 'assistant' : msg.sender
-          })),
+          chat_history: chatHistory
+            .filter(msg => {
+              // Only include messages with valid structure
+              return msg &&
+                     msg.sender &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)